beamformer_coefficient_generator/BeamformerCoefficientTest.cu

```
@@ -13,27 +13,53 @@ float ts_diff(struct timespec first, struct timespec last)
 {
     float time_difference = (float) last.tv_sec - (float) first.tv_sec;
     long nanosec_difference = last.tv_nsec - first.tv_nsec;
-    time_difference += (float) nanosec_difference / 1e9f; //Should work if this is negative as well?
+    time_difference += (float) nanosec_difference / 1e9f;
     return time_difference;
 }
 
-BeamformerCoeffTest::BeamformerCoeffTest(float fFloatingPointTolerance, BeamformerCoeffTest::SteeringCoefficientKernel eKernelOption):
+BeamformerCoeffTest::BeamformerCoeffTest(float fFloatingPointTolerance, 
+        BeamformerCoeffTest::SteeringCoefficientKernel eKernelOption, 
+        BeamformerCoeffTest::SteeringCoefficientBitWidth eBitWidth):
     m_fFloatingPointTolerance(fFloatingPointTolerance),
-    m_ulSizeSteeringCoefficients(NR_SAMPLES_PER_CHANNEL*NR_CHANNELS * NR_STATIONS * NR_BEAMS * COMPLEXITY * sizeof(float)),
     m_ulSizeDelayValues(NR_STATIONS * NR_BEAMS * sizeof(struct delay_vals)),
```

Generally it's safer to do `sizeof(*variable_name)`.
That way if you change the type that you're referring to, say it becomes `struct delay_vals_and_weights` or something, then this code will just work. Otherwise you may have silent errors.

Not going to nit-pick, won't stop me from merging this one. But for future reference.